src/script/src/table.rs

```diff
@@ -140,12 +143,18 @@ impl ScriptsTable {
         Ok(())
     }
 
-    pub async fn find_script_by_name(&self, name: &str) -> Result<String> {
+    pub async fn find_script_by_name(&self, schema: &str, name: &str) -> Result<String> {
         // FIXME(dennis): SQL injection
         // TODO(dennis): we use sql to find the script, the better way is use a function
         //               such as `find_record_by_primary_key` in table_engine.
-        let sql = format!("select script from {} where name='{}'", self.name(), name);
+        let sql = format!(
+            "select script from {} where schema='{}' and name='{}'",
```

Comment:
I think `schema` should be made primary key as well. You can find the table creation request in `ScriptTable::new`.

Reply:
Right